src/content_delegate.js

```diff
@@ -242,50 +221,67 @@ ContentDelegate.prototype.handleDocketDisplayPage = async function () {
   const radioDateInputs = [...document.getElementsByTagName('input')].filter(
     input => input.name === 'date_from' && input.type === 'radio'
   );
-  if (radioDateInputs.length > 1) {
-    return;
-  };
+  if (radioDateInputs.length > 1) { return; };
+
+  this.recap.getAvailabilityForDocket(
```

Comment:
The repetition between this and the other call to `getAvailablilityForDocket` bugs me, but more importantly, I don't think this needs to be behind an API call. I'm totally comfy with our users once in a while getting to the create alert page before the docket is truly ready. In the best case, the server will already have the item, but sometimes the server is really busy and processing can take time. I think we can just show the "create alert" button immediately and have them refresh on CL.